lighthouse-core/report/html/renderer/util.js

```diff
@@ -65,9 +65,11 @@ class Util {
     // The proto process turns 'not-applicable' into 'not_applicable'. Correct this to support both.
```

Comment:
should update this comment both for the new value and to mention supporting old reports